tests/unit/download/excelScenarioDownload.spec.ts

```diff
@@ -1,6 +1,5 @@
+/* eslint-disable perfectionist/sort-imports */
```

Comment:
import order matters here because we need to import ExcelScenarioDownload after the mocks in order for the mocks to work. It seems to be necessary to disable the rule for the whole import block - just disabling for the line which reported the error (line 12) wasn't recognised as disabling a rule which was being violated.

Reply:
oh no...